### DISCUSSION TITLE
Mantine UI v6?

### DISCUSSION THREAD

**Discussion by kitze** [mantinedev/mantine#5749] [Q&A] [answered]
I'm planning to keep my apps to use Mantine v6. Is there a hosted page for the Mantine UI components but with code snippets for v6?

Thank you!

**Comment by mariansimecek** [comment, marked as answer]
https://v6-ui-mantine.netlify.app/

**Comment by kitze** [comment]
@mariansimecek this is a lifesaver, thank you so much !!
